style/preview.less

```diff
@@ -344,6 +330,23 @@
 			}
 		}
 	}
+
+	&.dark {
```

Comment:
I wondering if there's a way to not repeat the dark CSS here and in the media query. Something like this https://stackoverflow.com/a/21819097 would make sense but not sure if possible with native CSS

Reply:
You're in luck! This is a less file and you can write [mixins](https://lesscss.org/features/#mixins-feature)

Reply:
This is what we need but for some reason it's not working for me yet, I'm not seeing CSS dark styles being applied at all

Reply:
Looking into whether it's something to do with our Vite set up, more investigation needed

Reply:
The mixin is working inside the media query but not inside the .dark class selector for some reason, trying to sort that out (haven't pushed yet just fyi)